frontend/src/mixins/apiPropsMixin.js

```diff
@@ -8,6 +8,7 @@ export const apiPropsMixin = {
     value: { required: false, default: null },
 
     /* field name and URI for saving back to API */
+    // TODO: consider renaming to 'path', as this now allows to contain a nested json path
```

Comment:
Ich würde eher ein Issue erstellen als einen TODO-Kommentar schreiben. Niemand sucht, kategorisiert oder updatet all die TODO-Kommentare im Code.

Reply:
Will be fixed with #2945